### ISSUE TITLE
Trouble restarting RF power after MP server shutdown/restart

### ISSUE THREAD

**Issue by r4m0n** [ReikaKalseki/Reika_Mods_Issues#754]
This one may be a bit hard to debug as I couldn't reproduce it in SP, but if you can't figure out what could cause it from the description, I can try setting a minimal server to try and isolate the problem.

General setup: (this side probably isn't the issue, but listed for reference) 16 Hydrokinetic Engines feeding a Rotational Dynamo (speed/torque properly balanced) directly feeding an RF Battery. The battery is connected (outputting to above) to hundreds of RF Cables, and ending mostly on Magnetostatic Engines of varying tiers.

Everything works well initially after connection, but the moment there is a server restart, I come back to all engines unpowered, and need to disconnect and reconnect the RF Cable at the battery to get everything working again (sorry, it occurred only about 3~4 times so far but on 100% of the server restarts, so I haven't had too many chances to experiment yet, so it may just need any change on the RF network to rebuild it).

I've also tried it with both a redstone signal at the battery and without, both cause the problem.

A last detail, the location is chunkloaded with a chickenchunks loader, so when the server starts the location gets loaded with no players in it. I've tried a test on SP with a chunkloader and everything works fine, so it probably isn't just that, but it may play a role in the problem.

**Comment by ReikaKalseki** [comment]
Something is probably off about the network building. Try various arrangements.

**Comment by r4m0n** [comment]
Good news, got it to show the same symptoms in SP and got a server crash while trying to put the engine working again.

Crash log: http://pastebin.com/LtVZJyNd

Setup: battery sitting in a chunk with a chunkloader, cables extending to an engine OUTSIDE the chunkloaded region (not what's happening on the server, but I'm guessing it may be chunk loading order on server startup?).

The battery is left running the engine and I leave the area, letting the engine chunk unload. When I return and reload the chunk, the engine remains dry. Checking the network transmission settings on both sides of the chunk barrier looked normal, and changing the settings doesn't bring the network to power the engine. When I broke a cable to reset the network (it was near the border area, but I think it was on the previously loaded region) I got the server to crash.
